Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.cpp

```diff
@@ -185,7 +186,8 @@ class UserMediaCaptureManagerProxySourceProxy final
     {
         ASSERT(isMainRunLoop());
 
-        m_pendingAction = m_pendingAction->isResolved() ? action() : m_pendingAction->whenSettled(RunLoop::main(), WTFMove(action));
+        Ref pendingAction = m_pendingAction;
```

Comment:
It would be a little more efficient to use:
```
Ref pendingAction = WTFMove(m_pendingAction);
```
We're about to reassign m_pendingAction on the next line so we don't care about leaving it in a good state and we can just steal it's "ref" to avoid refcounting churn.

Reply:
Fixed!